frontend/chartview.cpp

```diff
@@ -87,9 +94,16 @@ void ChartView::mouseReleaseEvent(QMouseEvent *event)
     {
         QApplication::restoreOverrideCursor();
     }
+    if (event->button() == Qt::LeftButton)
+    {
+        coordLabelX_->setText(NULL);
+        coordLabelY_->setText(NULL);
```

Comment:
Do these functions really take a `nullptr` to remove the text?  Not an empty `QString`?